### ISSUE TITLE
prody index 99999

### ISSUE THREAD

**Issue by maicol93** [prody/ProDy#793]
The most recent version of prody (1.10.7) is not able to parse pdb files with a high number of atoms restating the error:

Traceback (most recent call last):
  File "cluster.py", line 65, in <module>
    main()
  File "cluster.py", line 15, in main
    structure = parsePDB(sys.argv[1])
  File "/usr/local/lib/python2.7/dist-packages/prody/proteins/pdbfile.py", line 104, in parsePDB
    return _parsePDB(pdb[0], **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/prody/proteins/pdbfile.py", line 188, in _parsePDB
    result = parsePDBStream(pdb, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/prody/proteins/pdbfile.py", line 261, in parsePDBStream
    _parsePDBLines(ag, lines, split, model, chain, subset, altloc)
  File "/usr/local/lib/python2.7/dist-packages/prody/proteins/pdbfile.py", line 535, in _parsePDBLines
    charges[acount] = 0
IndexError: index 99999 is out of bounds for axis 0 with size 99999

the same error does not occur in previous versions (1.9.4)

**Comment by SHZ66** [comment]
Hi, this limit has been implemented since versions earlier than 1.9.4. There might be other bugs that cause this to show up. But anyway please first try:

```
ag = AtomGroup()
ag.setBetas([0.]*10000000)   # replace 10000000 with larger number if needed
structure = parsePDB(sys.argv[1], ag=ag)
```

**Comment by maicol93** [comment]
I just tried how you suggested me. version 1.10.7 of ProDy works as long as you indicate exactly the number of atoms that the system contains:

ag = AtomGroup()
        ag.setBetas([0.]*143478) 
        structure = parsePDB(sys.argv[1], ag=ag)

**Comment by SHZ66** [comment]
Thank you for verifying it. Yes, for it to work you may need to provide the exact number of atoms which is inconvenient. I added an argument to the parsePDB function (#794) so in the future, it is possible to change the upper limit of the number of atoms when loading large PDB files.

**Comment by jamesmkrieger** [comment]
We now have an actual fix for this (#825).
